diff -ur neon-0.29.6/configure neon-0.29.6.new/configure
--- neon-0.29.6/configure	2011-05-03 14:25:31.000000000 +0200
+++ neon-0.29.6.new/configure	2012-08-20 13:45:25.665811388 +0200
@@ -7201,6 +7201,9 @@
    NEON_LIBS=$ne_save_LIBS
 fi
 
+echo "Settings LDFLAGS TO $MINITAGE_LDFLAGS"
+NEON_LIBS="$MINITAGE_LDFLAGS -lssl -lcrypto -lexpat -lz"  
+export NEON_LIBS   
 
 # Check whether --with-libproxy was given.
 if test "${with_libproxy+set}" = set; then :
@@ -14520,7 +14523,7 @@
 
        CPPFLAGS="$CPPFLAGS -I$ne_expdir"
        if test "x${NEON_TARGET}" = "xlibneon.la"; then
-         NEON_LTLIBS=$with_expat
+         NEON_LTLIBS="$LD_FLAGS -lexpat -lxml"
        else
          # no dependency on libexpat => crippled libneon, so do partial install
          ALLOW_INSTALL=lib
@@ -15325,6 +15328,9 @@
 fi
 
 
+echo "Settings LDFLAGS TO $MINITAGE_LDFLAGS"
+NEON_LIBS="$MINITAGE_LDFLAGS -lssl -lcrypto -lexpat -lz"  
+export NEON_LIBS  
 # Pass the interface version on to libtool when linking libneon.la
 NEON_LINK_FLAGS="-version-info ${NE_LIBTOOL_VERSINFO}"
 # If any non-default ABI variations are used, add them to the SONAME:
